include/verilated_fst_sc.cpp

```diff
@@ -27,6 +27,17 @@
 //======================================================================
 //======================================================================
 
+void VerilatedFstSc::open(const char* filename) {
+    if (!sc_core::sc_get_curr_simcontext()->elaboration_done()) {
+        std::cerr << "VerilatedFstSc::open(\"" << filename
```

Comment:
```suggestion
        std::cerr << "%Error: VerilatedFstSc::open(\"" << filename
```

But perhaps vl_fatal is better.